{"uuid": "95f2d2a8-5113-4eaa-85c0-70398ee4842f", "children": ["b3a12033-3a14-44d6-93a5-5f1106a0328d", "8dd4ced0-0901-4b0d-996c-23525dd9c15a", "0454ca85-c38d-40f4-863e-fbd0f77471a3", "f8fffd52-d85a-455a-a55f-0e31cf7e0508", "c795a1de-aa28-4316-927f-960ae23de25a", "9405e8ba-310d-4abb-bfb2-5df60b15cf72", "1094a578-b984-44cc-9c93-e23b05f1af38", "c4d5924f-aa95-4cec-bd11-dfc4d93c4a73", "4c010ce3-2469-447a-af84-ac9a88a4faf4", "52126110-ffd8-4033-a7e0-577cb67a8afe", "29a67af7-a337-4925-916a-f29441f66833", "afa9ab15-90ba-4805-9a78-6143e97595e2"], "befores": [{"name": "_unittest_setUpClass_fixture_LoginTest", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: unknown error: Microsoft Edge failed to start: crashed.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from msedge location C:\\Program Files (x86)\\Microsoft\\Edge\\Application\\msedge.exe is no longer running, so msedgedriver is assuming that msedge has crashed.)\nStacktrace:\nBacktrace:\n\tGetHandleVerifier [0x00007FF76F63B5A2+64226]\n\tMicrosoft::Applications::Events::EventProperty::~EventProperty [0x00007FF76F5CBB62+770994]\n\t(No symbol) [0x00007FF76F38CC3C]\n\t(No symbol) [0x00007FF76F3BB838]\n\t(No symbol) [0x00007FF76F3B65EA]\n\t(No symbol) [0x00007FF76F3F6D2A]\n\t(No symbol) [0x00007FF76F3EEDC3]\n\t(No symbol) [0x00007FF76F3C3BDC]\n\t(No symbol) [0x00007FF76F3C2DC6]\n\t(No symbol) [0x00007FF76F3C4354]\n\tMicrosoft::Applications::Events::ILogManager::DispatchEventBroadcast [0x00007FF76F818E49+1319081]\n\t(No symbol) [0x00007FF76F43BEE8]\n\tMicrosoft::Applications::Events::EventProperty::~EventProperty [0x00007FF76F517AC1+33553]\n\tMicrosoft::Applications::Events::EventProperty::~EventProperty [0x00007FF76F50FEFF+1871]\n\tMicrosoft::Applications::Events::ILogManager::DispatchEventBroadcast [0x00007FF76F817A63+1313987]\n\tMicrosoft::Applications::Events::ILogConfiguration::operator* [0x00007FF76F5D4198+20232]\n\tMicrosoft::Applications::Events::ILogConfiguration::operator* [0x00007FF76F5D07B4+5412]\n\tMicrosoft::Applications::Events::ILogConfiguration::operator* [0x00007FF76F5D08AC+5660]\n\tMicrosoft::Applications::Events::EventProperty::~EventProperty [0x00007FF76F5C4561+740785]\n\tBaseThreadInitThunk [0x00007FF965347614+20]\n\tRtlUserThreadStart [0x00007FF9662626A1+33]\n\n", "trace": "  File \"F:\\Sivakumar\\Pytest Automation\\PycharmProjects\\POMFramework\\venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"F:\\Sivakumar\\Pytest Automation\\PycharmProjects\\POMFramework\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1130, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"F:\\Sivakumar\\Pytest Automation\\PycharmProjects\\POMFramework\\venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 902, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"F:\\Sivakumar\\Pytest Automation\\PycharmProjects\\POMFramework\\venv\\Lib\\site-packages\\_pytest\\unittest.py\", line 157, in fixture\n    setup()\n  File \"F:\\Sivakumar\\Pytest Automation\\PycharmProjects\\POMFramework\\TestCases\\test_Base_1.py\", line 26, in setUpClass\n    cls.driver = webdriver.Edge(options=cls.options, service=EdgeService(EdgeChromiumDriverManager().install()))\n                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"F:\\Sivakumar\\Pytest Automation\\PycharmProjects\\POMFramework\\venv\\Lib\\site-packages\\selenium\\webdriver\\edge\\webdriver.py\", line 76, in __init__\n    super().__init__(\n  File \"F:\\Sivakumar\\Pytest Automation\\PycharmProjects\\POMFramework\\venv\\Lib\\site-packages\\selenium\\webdriver\\chromium\\webdriver.py\", line 104, in __init__\n    super().__init__(\n  File \"F:\\Sivakumar\\Pytest Automation\\PycharmProjects\\POMFramework\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 286, in __init__\n    self.start_session(capabilities, browser_profile)\n  File \"F:\\Sivakumar\\Pytest Automation\\PycharmProjects\\POMFramework\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 378, in start_session\n    response = self.execute(Command.NEW_SESSION, parameters)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"F:\\Sivakumar\\Pytest Automation\\PycharmProjects\\POMFramework\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 440, in execute\n    self.error_handler.check_response(response)\n  File \"F:\\Sivakumar\\Pytest Automation\\PycharmProjects\\POMFramework\\venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1686133497322, "stop": 1686133504030}], "start": 1686133497322, "stop": 1686133504636}